.travis.yml

```diff
@@ -4,9 +4,11 @@ php:
     - 7.0
     - 7.1
     - 7.2
+    - 7.3
+    - 7.4
 
 install:
-    - composer install
+    - composer update
```

Comment:
Why?

Reply:
Once the `composer.lock` is removed from Git version control, this can be reverted.

Reply:
Let's do it in this PR?

Reply:
@umpirsky, I've done for this. You can look at latest changes :).